backtracking/power_sum.py

```diff
@@ -0,0 +1,77 @@
+"""
+Problem source: https://www.hackerrank.com/challenges/the-power-sum/problem
+Find the number of ways that a given integer X, can be expressed as the sum
+of the Nth powers of unique, natural numbers. For example, if X=13 and N=2.
+We have to find all combinations of unique squares adding up to 13.
+The only solution is 2^2+3^2. Constraints: 1<=X<=1000, 2<=N<=10.
+"""
+
+from math import pow
+
+sum_ = count_ = 0
+
+
+def backtrack(x: int, n: int, i: int) -> None:
```

Comment:
Give descriptive name for x,n etc....like num, value.

Reply:
Improved paramater name in d4e18f1